unit_tests/test_plugins.py

```diff
@@ -96,33 +96,51 @@ def test_install_do_remove_unlisted(self, mock_get_plugins_to_install, mock_inst
         are removed from disk.
         """
         plugin_name = "plugin"
+        dependency_plugin_name = "dependecy"
         plugin_path = os.path.join(paths.PLUGINS, "{}-1.jpi".format(plugin_name))
-        mock_get_plugins_to_install.return_value = {plugin_name}
+        plugin_list = "plugin listed"
+
+        def side_effect(value):
+            if value == plugin_list.split():
+                fake_result = plugin_list.split()
+                fake_result.append(dependency_plugin_name)
+                return fake_result
+            return {plugin_name, "dependecy"}
```

Comment:
Obviously this'll need changing as well per above

Reply:
addressed in a09e7bb